jupyterlab_git/handlers.py

```diff
@@ -75,7 +105,8 @@ async def post(self):
 
         show_top_level = await self.git.show_top_level(current_path)
         if show_top_level["code"] != 0:
```

Comment:
```suggestion
        if show_top_level.get("top_repo_path") is None:
```

This handler needs to be removed (it is not called anymore). So we should continue raising an error to be sure the other missing data are not wrongly handled.

Reply:
Thanks, I'll revert this week!

Reply:
ahh just to say I didn't manage this week and will be away for a few days, but will be able so sit on this next weekend 👍